app/qml/map/MMMapController.qml

```diff
@@ -800,7 +800,13 @@ Item {
 
     drawerHeader.title: qsTr( "Choose Active Layer" )
 
```

Comment:
There is one thing that I realized while reviewing the PR is that we need to instantiate this drawer dynamically (wrap it in Loader/Component) - the reason is that it holds references (pointers) to the opened qgs project and layers, and in case we open a different project, we would need to invalidate all the pointers. It will be easier to have this drawer dynamically instantiated only when requested.